templates/guest_book/gulpfilelocal.js

```diff
@@ -0,0 +1,120 @@
+const gulp = require("gulp");
+const fs = require('fs');
+const args = require("yargs").argv;
+const { Account, SimpleKeyStoreSigner, InMemoryKeyStore, KeyPair, LocalNodeConnection, NearClient, Near } = require('nearlib');
+
+gulp.task('copyfiles', function () {
+  return gulp.src('./assembly/**/*')
+    .pipe(gulp.dest('./out/'));
+});
+
+gulp.task("build:model", callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "model.ts",
+    "--baseDir", "./out",
+    "--nearFile", "../out/model.near.ts",
+    "--measure"
+  ], callback);
+});
+
+gulp.task("build:bindings", ["build:model"], callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "main.ts",
+    "--baseDir", "./out",
+    "--binaryFile", "../out/main.wasm",
+    "--nearFile", "../out/main.near.ts",
+    "--measure"
+  ], callback);
+});
+
+gulp.task("build:all", ["build:bindings"], callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "../out/main.near.ts",
+    "--baseDir", "./out",
+    "-O3",
+    "--binaryFile", "../out/main.wasm",
+    "--sourceMap",
+    "--measure"
+  ], callback);
+});
+
+gulp.task('build', function(done) {
+  const runSequence = require('run-sequence');
+  runSequence('copyfiles', 'build:all', function() {
+    done();
+  });
+});
+
+gulp.task('deploy', function(done) {
+  // Find keys/ account id
+  const keyPath = './neardev/devkey.json';
+  if (!fs.existsSync(keyPath)) {
+    console.error("Deployment failed. File ./neardev/devkey.json does not exist. Please add this file and try again. Please make sure file contains public_key, secret_key, and account_id.")
```

Comment:
> Please add this file and try again

It's probably good idea to explain how to generate this file / link to docs.

Reply:
I am planning to generate this file automatically as another cli script

Reply:
(but yes, docs will also be there)